README.md

```diff
@@ -47,15 +47,16 @@ pip install astro-sdk-python[amazon,google,snowflake,postgres]
     airflow db init
     ```
 
-   > **Note:** `AIRFLOW__CORE__ENABLE_XCOM_PICKLING` no longer needs to be enabled from `astro-sdk-python` release 1.2 and above. This functionality is now deprecated as our custom xcom backend handles serialization.
-    Users can either use a custom XCom backend [`AstroCustomXcomBackend`](https://astro-sdk-python.readthedocs.io/en/latest/guides/xcom_backend.html) with Xcom pickling disabled (or) enable Xcom pickling.
-    Currently, custom XCom backends are limited to data types that are json serializable. Since Dataframes are not json serializable, we need to enable XCom pickling to store dataframes.
+   > **Note:** `AIRFLOW__CORE__ENABLE_XCOM_PICKLING` no longer needs to be enabled from astro-sdk-python release 1.2 and above. This functionality is now deprecated.
+   > - For airflow version < 2.5 and astro-sdk-python release < 1.2 Users can either use a custom XCom backend [AstroCustomXcomBackend](https://astro-sdk-python.readthedocs.io/en/latest/guides/xcom_backend.html#xcom-backend) with Xcom pickling disabled (or) enable Xcom pickling. Currently, custom XCom backends are limited to data types that are json serializable. Since Dataframes are not json serializable, we need to enable XCom pickling to store dataframes.
```

Comment:
>Currently, custom XCom backends are limited to data types that are json serializable. Since Dataframes are not json serializable, we need to enable XCom pickling to store dataframes.

This is not true for Custom XCom Backend !

We create Custom XCom backend to handle more types. Check https://github.com/astronomer/astro-sdk/blob/main/python-sdk/src/astro/custom_backend/serializer.py

Reply:
Removed.